data_model/src/asset.rs

```diff
@@ -38,6 +40,11 @@ pub type AssetDefinitionsMap =
 pub type AssetTotalQuantityMap =
     btree_map::BTreeMap<<AssetDefinition as Identifiable>::Id, NumericValue>;
 
+/// [`AssetRegisterTxMap`] provides an API to work with collection of key([`AssetId`])-value([`HashOf<VersionedSignedTransaction>`])
+/// pairs.
+pub type AssetRegisterTxsMap =
+    btree_map::BTreeMap<<Asset as Identifiable>::Id, HashOf<VersionedSignedTransaction>>;
```

Comment:
In PR description you write about `DashMap`

Reply:
Just remove the reference to the concrete type used in the implementation. Anyone interested enough in this can look it up in the text of the program.